### PR TITLE
Update translation (DE/JP)

### PR DESCRIPTION
**Pull request by luixxiul**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan: n/a

Auditors: @aekeus

### CODE REVIEW

File: app/extensions/brave/locales/ja-JP/preferences.properties [hunk 1]
```diff
@@ -17,9 +17,9 @@ paymentsWelcomeText1=Braveでは、よく訪問するサイトに自動で寄付
 paymentsWelcomeText2=Braveペイメントの使用を開始するには、このページの上にあるスイッチを切り替えてください。後は簡単です。
 paymentsWelcomeText3=Braveペイメントは、ユーザーが広告をブロックしても、サイトのパブリッシャーが事業を続けられるようにするシステムです。これは台帳システムを通じて、ユーザーの資金をサイトの所有者に送金するものです。台帳システムは匿名なので、訪問したサイトや寄付を行ったサイトからユーザーが特定されることはありません。
 paymentsWelcomeText4=Braveペイメントは現在ベータ版です。ユーザーの方々のフィードバックをもとに、システムの微調整を行いたいと考えています。
-paymentsWelcomeText5=注: Braveペイメントでは、ユーザーの所在地に基づき、ウォレットの資金調達に最適なオプションを提供するため、国検索サービスを行っています。このサービスは完全に匿名で利用できます。
-paymentsWelcomeText6=詳細
-paymentsWelcomeText7=Braveペイメントに…
+paymentsWelcomeText5=注: Braveペイメントでは、国検索サービスで検出されたユーザーの所在地に基づき、ウォレットへの入金に最適なオプションを提供しています。この検索サービスは完全に匿名で機能します。
+paymentsWelcomeText6=詳細は次のリンクからご確認ください。
+paymentsWelcomeText7=for Brave Payments…
```
Comment (luixxiul) [inline app/extensions/brave/locales/ja-JP/preferences.properties]: Revered due to this https://github.com/brave/browser-laptop/issues/4102

File: app/extensions/brave/locales/ja-JP/menu.properties [hunk 2]
```diff
@@ -48,7 +48,7 @@ clearHistory=履歴を消去
 clearCache=キャッシュを消去
 clearSiteData=全てのクッキーとサイトのデータを消去
 recentlyClosed=最近閉じたタブ
-recentlyVisited=最近訪れたサイト
+recentlyVisited=Recently Visited
```
Comment (luixxiul) [inline app/extensions/brave/locales/ja-JP/menu.properties]: Reverted due to that this is TODO

**Comment by bbondy** [comment]
I'm not sure if there is any point merging this because the next l10n update will break it again no? Can we fix in transifex?

**Comment by luixxiul** [comment]
@bbondy I'm maintaining them on transifex so no worry

**Comment by bbondy** [comment]
ok will merge.

**Pull request merged**